src/quickpicks/schemas.ts

```diff
@@ -106,12 +106,11 @@ export async function schemaVersionQuickPick(
   subject: string,
 ): Promise<Schema | undefined> {
   const loader = ResourceLoader.getInstance(schemaRegistry.connectionId);
-  const schemaVersions: Schema[] = await loader.getSchemasForEnvironmentId(
```

Comment:
This call to getSchemas...() snuck in at the last minute, concurrent with prior branch aiming to have gotten rid of all callers. No biggie, now gone.